package.json

```diff
@@ -38,6 +41,7 @@
   },
   "dependencies": {
     "axios": "^1.4.0",
+    "eslint-plugin-jsdoc": "^46.8.2",
```

Comment:
Why is it not a dev dependency?

Reply:
whoops good catch
https://github.com/stellar/typescript-wallet-sdk/pull/72/commits/038808837b722268e69d421cfbab4fba91eca682